src/main/server/enable_ssl.sh

```diff
@@ -135,6 +136,25 @@ if [ ! -d "$INSTALL_DIRECTORY" ]; then
   exit 1
 fi
 
+freedomain(){
+  env=$(<.env)
+  hostname="ams-$RANDOM"
+  if [ `cat $INSTALL_DIRECTORY/conf/red5.properties | egrep "rtmps.keystorepass=ams-[0-9]*.antmedia.cloud"|wc -l` == "0" ]; then
+    ip=`curl http://checkip.amazonaws.com`
+    curl -X POST -H "x-api-key: $env" -H "Content-Type: application/json" "https://emgwmames0.execute-api.eu-central-1.amazonaws.com/ams/create?domain=$hostname&ip=$ip"
```

Comment:
I think it is better to use cname instead of emgwmames0.execute-api.eu-central-1.amazonaws.com.
Secondly, adding license key to the url would be better to authenticate the requests

Reply:
already added.